etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/ExamplesHelper.groovy

```diff
@@ -0,0 +1,26 @@
+package gov.hhs.cdc.trustedintermediary
+
+import gov.hhs.cdc.trustedintermediary.context.ApplicationContext
+import gov.hhs.cdc.trustedintermediary.external.hapi.HapiFhirImplementation
+import gov.hhs.cdc.trustedintermediary.external.hapi.HapiFhirResource
+import org.hl7.fhir.r4.model.Bundle
+
+import java.nio.file.Files
+
+class ExamplesHelper {
```

Comment:
Java docs

Reply:
This is a unit test file, so while we don't require them, if something is strange and can benefit explaining "why", then please add JavaDocs.